services/workflows-service/prisma/migrations/20240731131954_add_ui_definition_naming/migration.sql

```diff
@@ -0,0 +1,6 @@
+-- AlterTable
+ALTER TABLE "UiDefinition" ADD COLUMN     "displayName" TEXT,
+ADD COLUMN     "name" TEXT;
+
+-- CreateIndex
+CREATE INDEX "UiDefinition_name_projectId_idx" ON "UiDefinition"("name", "projectId");
```

Comment:
didnt you say you wanted to add it afterwards? what would happen with empty names in current envs?

Reply:
it's an index not unique index, it's for searching, etc. later on we'll alter and make it unique index